### ISSUE TITLE
one service - two shown in the table

### ISSUE THREAD

**Issue by dovholuknf** [openziti/desktop-edge-ui#114]
i was on the identity page when i removed and added a service back in quick succession. you can see the count shown is correct (1) but there are two entries in the table to the right

![image](https://user-images.githubusercontent.com/46322585/232232019-cc441f54-12a6-4663-9a61-757a74d0891f.png)

**Comment by dovholuknf** [comment]
easily reproducible. 7 are shown in the table here, one count:
![image](https://user-images.githubusercontent.com/46322585/232232151-99c86e4b-7bd3-42b2-b2a8-c69c3b7145cf.png)

**Comment by dovholuknf** [comment]
bash commands i was running:
```
svc=docker.whale
intercept_address="${svc}.ziti"
intercept_port=80
offload_address=localhost
offload_port=2000

function createService {
  ziti edge create config ${svc}.host.config host.v1 '{"protocol":"tcp", "address":"'"${offload_address}"'", "port":'${offload_port}'}'
  ziti edge create config ${svc}.int.config  intercept.v1 '{"protocols":["tcp"],"addresses":["'"${intercept_address}"'"], "portRanges":[{"low":'${intercept_port}', "high":'${intercept_port}'}]}'
  ziti edge create service "${svc}" --configs "${svc}.host.config","${svc}.int.config"
  ziti edge create service-policy "${svc}.bind" Bind --service-roles "@${svc}" --identity-roles "#${svc}.binders"
  ziti edge create service-policy "${svc}.dial" Dial --service-roles "@${svc}" --identity-roles "#${svc}.dialers"
}

function deleteService {
  ziti edge delete config  where 'name contains "'"${svc}"'."'
  ziti edge delete service where 'name = "'"${svc}"'"'
  ziti edge delete sp      where 'name contains "'"${svc}"'."'
}
```

then run this and every time the ui paints, re-run it and the table will keep growing
```
deleteService; createService
```

**Comment by JeremyTellier** [comment]
Fixed in 3.2.8
